proto/sentium/api/v1/principals.proto

```diff
@@ -5,33 +5,39 @@ package sentium.api.v1;
 import "google/api/annotations.proto";
 import "google/protobuf/struct.proto";
 
+// The service Principals manages Principals within Sentium
 service Principals {
+	// Creates a principal
 	rpc Create(PrincipalsCreateRequest) returns (Principal) {
 		option (google.api.http) = {
 			post : "/v1/principals"
 			body : "*"
 		};
 	}
 
+	// Deletes a principal
 	rpc Delete(PrincipalsDeleteRequest) returns (PrincipalsDeleteResponse) {
 		option (google.api.http) = {
 			delete : "/v1/principals/{id}"
 			body : "*"
 		};
 	}
 
+	// Lists principals
 	rpc List(PrincipalsListRequest) returns (PrincipalsListResponse) {
 		option (google.api.http) = {
 			get : "/v1/principals?segment={segment}&_limit={pagination_limit}&_start={pagination_token}"
 		};
```

Comment:
This is a strange way to do this. It makes the openapi generater have a heart attack...

I would suggest not to map variables like this 😬

Reply:
🤔 which generator did you try? I haven't actually tried this using a proxy like Envoy either so this might be a no go anyway.

Reply:
https://pkg.go.dev/github.com/google/gnostic/cmd/protoc-gen-openapi